test/integration/controller/jobset_controller_test.go

```diff
@@ -761,7 +761,7 @@ func matchJobsNodeSelectors(js *jobset.JobSet, nodeSelectors map[string]map[stri
 		jobsUpdated++
 	}
 	// Calculate expected number of updated jobs
-	wantJobsUpdated := 0
+	var wantJobsUpdated int32 = 0
 	for _, rjob := range js.Spec.ReplicatedJobs {
 		if _, exists := nodeSelectors[rjob.Name]; exists {
 			wantJobsUpdated += rjob.Replicas
```

Comment:
```suggestion
			wantJobsUpdated += int(rjob.Replicas)
```
I think it would be better to stick to this same pattern used in the other functions `numJobsExpectedToSucceed()` and `NumExpectedJobs()`.

Looks good to me after this!

Reply:
updated.